public/locales/en/common.json

```diff
@@ -176,24 +176,49 @@
     },
     "buttons": {
       "edit-back": "Back to play",
-      "start-position": "Start",
-      "end-position": "End",
-      "click-to-set": "Click to set",
       "tap-to-unmute": "TAP TO UNMUTE"
     },
-    "share": {
-      "title": "watch this video"
-    },
     "error": {
       "loading": "Error loading file."
     },
     "header": {
       "series-by-topic": "Series of lessons on the topic:"
     },
     "download": {
-      "iconHoverText": "Download the excerpt you marked",
-      "downloadButton": "Download",
+      "icon-hover-text": "Download the excerpt you marked",
+      "download-button": "Download",
       "copyButton": "Copy link"
+    },
+    "settings": {
```

Comment:
Are all those properly updated in all languages?

Reply:
yes, I was add it on all lalnguages